app/src/main/java/com/kylecorry/trail_sense/astronomy/infrastructure/receivers/SunsetAlarmReceiver.kt

```diff
@@ -70,5 +70,22 @@ class SunsetAlarmReceiver : BroadcastReceiver() {
                 start(fragment.requireContext())
             }
         }
+
+        fun disable(
+            fragment: Fragment
+        ) {
+            val preferences = UserPreferences(fragment.requireContext())
+            preferences.astronomy.sendSunsetAlerts = false
+
+            preferences.astronomy.isSunsetAlertEnabled = false
+
+            val scheduler = scheduler(fragment.requireContext())
+            scheduler.cancel()
+        }
+
+        fun isSunsetAlertEnabled(context: Context): Boolean {
```

Comment:
Can you remove this method? It isn't used and I prefer retrieving that from the UserPreferences because it is only the sendSunsetAlerts that controls it